src/index.html

```diff
@@ -17,6 +17,54 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="link-logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="moyo logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li>
+            <a
+              href="#"
+              class="is-active"
+            >
+              Apple
```

Comment:
Don't use spaces in `<a>` tag's `href` property. Anchor links should start with the `#` symbol followed by the link target. For example, `href="#apple"` if you're linking to an element with the ID 'apple'.